ip_repo/amdc_eddy_current_sensor_1.0/README.md

```diff
@@ -20,8 +21,9 @@ The IP is accessed via the AXI4-Lite register-based interface from the DSP. This
 | -- | -- | -- | -- |
 | 0x00 | SENSOR_DATA_X | R | IP DATA REGISTER |
 | 0x04 | SENSOR_DATA_Y | R | IP DATA REGISTER |
```

Comment:
This is pedantic, but can you change `IP DATA REGISTER` to be same word caps as your config regs